backend/jottacloud/replace.go

```diff
@@ -0,0 +1,84 @@
+/*
+Translate file names for JottaCloud adapted from OneDrive
```

Comment:
Can you copy and then adapt the tests from backent/onedrive/replace_test.go too please.

Reply:
Will do.